components/table/rowItem.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Meta } from "@storybook/react";
+import { RowItem } from "./RowItem";
+import { Tag, Input, Space } from "antd";
+import { EditOutlined, DeleteOutlined } from "@ant-design/icons";
+
+const meta: Meta = {
+  title: "Atoms/RowItem",
+  component: RowItem,
+  tags: ["autodocs"],
+  argTypes: {
+    children: {
+      control: "text",
+      description: "Content to display inside the row item",
+    },
+  },
+} satisfies Meta<typeof RowItem>;
```

Comment:
Use `as` – that works better than `satisfies`.
```suggestion
} as Meta<typeof RowItem>;
```

Reply:
Done